### PR TITLE
Image.getRawData method

### PR DESCRIPTION
**Pull request by basvanmeurs**
Background info: WPE UI Framework depends on node-canvas for creating text bitmaps, rounded edges and image parsing.

When you need to read parsed raw ARGB image data, you need to load the image, then draw it on a canvas and convert it to a buffer:

```javascript
        var img = new Canvas.Image();
        img.src = data;
        var canvas = new Canvas(img.width, img.height);
        var ctx = canvas.getContext('2d');
        ctx.drawImage(img, 0, 0, img.width, img.height);
        var buffer = canvas.toBuffer('raw');
```

This is an unnecessary double copy of the data. This pull request contains a new method: Image.getRawData, which returns the buffer directly:
```javascript
        var img = new Canvas.Image();
        img.src = data;
        var buffer = img.getRawData();
```
This allows for a more efficient usage of Canvas.Image.

We'd appreciate it if you'd merge this into the node-canvas module!

### CODE REVIEW

File: lib/context2d.js [hunk 1]
```diff
@@ -16,10 +16,6 @@ var canvas = require('./bindings')
   , CanvasPattern = canvas.CanvasPattern
   , ImageData = canvas.ImageData;
 
-var parseCssFont = require('parse-css-font');
```
Comment (LinusU) [inline lib/context2d.js]: The removal of this should be it's own PR, it also should remove the dependencies from the package.json...

Comment (basvanmeurs) [inline lib/context2d.js]: I didn't realise that it would be part of the pull request.. that was not intentional.

**Comment by zbjornson** [comment]
@basvanmeurs if you still want this to be merged, can you clean up the branch so it contains only the relevant changes please per @LinusU's feedback?

**Comment by zbjornson** [comment]
(I guess this is replaced by #1256)

**Comment by basvanmeurs** [comment]
Correct, it can be closed ;-) Thanks zbjornson!
